src/default-configuration.ts

```diff
@@ -10,4 +10,5 @@ export const defaultAppConfiguration: AppConfiguration = {
     "https://cdn.fromdoppler.com/unlayer-editor/asset-manifest-v2.json",
   loaderUrl: "https://cdn.fromdoppler.com/loader/v1/loader.js",
   unlayerProjectId: 32092,
+  dopplerLegacyBaseUrl: "https://appint.fromdoppler.net",
```

Comment:
Remember to create a new PR in Swarm repository with the configuration for each environment, you should update the [HTML template](https://github.com/MakingSense/doppler-swarm/blob/master/webapps-stack/editors/index.template.html) and the environment files

Reply:
Its Ok, https://github.com/MakingSense/doppler-swarm/blob/0019fee3689b17c4e24f8d9a93a4bc7abb4e5054/webapps-stack/editors/index.template.html#L60

Reply:
Oh, I forgot it hehe, thanks!